### PR TITLE
Indian states added to en_IN locale

### PR DESCRIPTION
**Pull request by jiveshsg**
Indian states added to en_IN locale

### CODE REVIEW

File: src/Faker/Provider/en_IN/Address.php [hunk 1]
```diff
@@ -28,6 +27,14 @@ class Address extends \Faker\Provider\Address
         'Warangal',
     );
 
+    protected static $state = array(
+        'Andhra Pradesh', 'Arunachal Pradesh', 'Assam', 'Bihar', 'Chhattisgarh', 'Goa', 'Gujarat', 'Haryana', 'Himachal Pradesh', 'Jammu and Kashmir', 'Jharkhand', 'Karnataka', 'Kerala', 'Madhya Pradesh', 'Maharashtra', 'Manipur', 'Meghalaya', 'Mizoram', 'Nagaland', 'Odisha', 'Punjab', 'Rajasthan', 'Sikkim', 'Tamil Nadu', 'Telangana', 'Tripura', 'Uttar Pradesh', 'Uttarakhand', 'West Bengal', 'Andaman and Nicobar Islands', 'Chandigarh', 'Dadra and Nagar Haveli', 'Daman and Diu', 'Delhi', 'Lakshadweep', 'Puducherry'
```
Comment (fzaninotto) [inline src/Faker/Provider/en_IN/Address.php]: please split the line by capital letter, and order alphabetically

Comment (jiveshsg) [inline src/Faker/Provider/en_IN/Address.php]: The administrative region equivalent of "states" in India consists of "states" plus "union territories", therefore the list is arranged alphabetically in two batches where the last 7 entries are union territories (form of states but governed by the central government, with exception of Delhi, i.e. the the capital as well as union territory, also has separate government). So this was the correct way of doing this in my opinion.

Also, please give an example of what do you mean by splitting the line by capital letter. Is is like the following?
```
'Andhra
Pradesh', ...
```

Comment (fzaninotto) [inline src/Faker/Provider/en_IN/Address.php]: Gotcha. For the line splitting, it's more like

```php
'Andhra Pradesh', 'Arunachal Pradesh', 'Assam',
'Bihar',
'Chhattisgarh',
'Goa', 'Gujarat',
...
```

**Comment by fzaninotto** [comment]
You can't use tabs for padding, see the Travis error report.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**